pkgs/applications/audio/muse/default.nix

```diff
@@ -1,33 +1,68 @@
-{ lib, stdenv, fetchFromGitHub, cmake, pkg-config, qttools, wrapQtAppsHook
-, alsa-lib, dssi, fluidsynth, ladspaH, lash, libinstpatch, libjack2, liblo
-, libsamplerate, libsndfile, lilv, lrdf, lv2, qtsvg, rtaudio, rubberband, sord, serd
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, pkg-config
+, qttools
+, wrapQtAppsHook
+, alsa-lib
+, dssi
+, fluidsynth
+, ladspaH
+, lash
+, libinstpatch
+, libjack2
+, liblo
+, libsamplerate
+, libsndfile
+, lilv
+, lrdf
+, lv2
+, qtsvg
+, rtaudio
+, rubberband
+, sord
+, serd
 }:
 
 stdenv.mkDerivation rec {
```

Comment:
Can we switch to [the `finalAttrs` / "passing a function instead of recursive set" pattern](https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes)?